docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html

```diff
@@ -52,8 +52,12 @@
 <hr>
 
 <blockquote>
+    <p><b>Note:</b> If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually
+        ( Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.)
```

Comment:
```suggestion
        Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
```

The two notes you have added should be consistent with each other.